models/articlesModel.js

```diff
@@ -60,4 +61,17 @@ const updateArticleVotes = (articleId, votes) => {
 		});
 };
 
-module.exports = { fetchArticles, fetchArticleById, updateArticleVotes };
+const addComment = (articleId, username, body) => {
+	return db
+		.query(
+			`INSERT INTO comments (body, votes, author, article_id)
```

Comment:
In your table creation, your comments have a DEFAULT value of 0. So we don't need include this as a column for inserting values. It will be done automatically